### PR TITLE
Update show-tooltip.md

### PR DESCRIPTION
**Pull request by minheq**
Consistent demo documentation

### CODE REVIEW

**Comment by zy410419243** [comment]
Update snapshots please.

**Comment by afc163** [comment]
Update snapshots please.

**Comment by zy410419243** [comment]
Run `npm test -- -u Slider` and push changed snapshots.

**Comment by minheq** [comment]
Ah ok! But it should not have changed snapshots 🤔

**Comment by minheq** [comment]
@zy410419243 I did these things locally:
1. `npm test` (all pass)
2. `npm test -- -u Slide` (nothing changed)
3. `npm run test-all` (same snapshot failure)

```
Received value does not match stored snapshot "renders ./components/slider/demo/show-tooltip.md correctly 1".

    - Snapshot
    + Received

    @@ -19,28 +19,9 @@
          class="ant-slider-handle ant-tooltip-open"
          role="slider"
          style="left:30%"
          tabindex="0"
        />
    -   <div>
    -     <div
    -       class="ant-tooltip  ant-tooltip-placement-top "
    -     >
    -       <div
    -         class="ant-tooltip-content"
    -       >
    -         <div
    -           class="ant-tooltip-arrow"
    -         />
    -         <div
    -           class="ant-tooltip-inner"
    -           role="tooltip"
    -         >
    -           30
    -         </div>
    -       </div>
    -     </div>
    -   </div>
        <div
          class="ant-slider-mark"
        />
      </div>

      56 |       ariaConvert(wrapper);
      57 |
    > 58 |       expect(wrapper).toMatchSnapshot();
         |                       ^
      59 |       MockDate.reset();
      60 |     });
      61 |   });
```

4. manually edit snapshot, remove the portion above. it complain that the portion is missing
```
Received value does not match stored snapshot "renders ./components/slider/demo/show-tooltip.md correctly 1".

    - Snapshot
    + Received

    @@ -19,9 +19,28 @@
          class="ant-slider-handle ant-tooltip-open"
          role="slider"
          style="left:30%"
          tabindex="0"
        />
    +   <div>
    +     <div
    +       class="ant-tooltip  ant-tooltip-placement-top "
    +     >
    +       <div
    +         class="ant-tooltip-content"
    +       >
    +         <div
    +           class="ant-tooltip-arrow"
    +         />
    +         <div
    +           class="ant-tooltip-inner"
    +           role="tooltip"
    +         >
    +           30
    +         </div>
    +       </div>
    +     </div>
    +   </div>
        <div
          class="ant-slider-mark"
        />
      </div>

      56 |       ariaConvert(wrapper);
      57 |
    > 58 |       expect(wrapper).toMatchSnapshot();
         |                       ^
      59 |       MockDate.reset();
      60 |     });
      61 |   });
```

I don't know how to fix this. can you help?

**Comment by zy410419243** [comment]
Try install `jest-cli` globally and re-run `npm test -- -u Slider`, may be helpful.

File: components/slider/demo/show-tooltip.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ title:
  When `tooltipVisible` is `true`, ToolTip will show always, or ToolTip will not show anyway, even if dragging or hovering.
```
Comment (zy410419243) [inline components/slider/demo/show-tooltip.md]: Maybe `even though` is better than `even if`?

**Comment by minheq** [comment]
Sorry I was not able to troubleshoot snapshot issues. Thanks for merging!

**Comment by afc163** [comment]
@minheq Actually it is our problem~

**Pull request merged**